### DISCUSSION TITLE
상품 이미지 다건 등록시의 URL 유효기간이 궁금합니다.

### DISCUSSION THREAD

**Discussion by importunate-dev** [commerce-api-naver/commerce-solutions#36] [묻고 답하기] [answered]
안녕하세요. 상품 이미지를 등록했을 때, 생성되는 링크의 유효기간이 궁금합니다.
/v1/product-images/upload
엔드포인트로 요청을 보내서 받은 링크는 언제까지 유효할까요?
해당 링크를 상품 상세페이지의 <img src> 태그에 넣었을 때 엑스박스가 언제 뜰지 궁금하여 질문 드립니다.

**Comment by commerce-api-naver** [comment, marked as answer]
안녕하세요. 커머스API 이민석입니다.

'[상품] 상품 이미지 다건 등록 (`POST /v1/product-images/upload`)' API로 등록된 이미지에 별도의 서비스 제공 유효기간을 정하고 있진 않습니다.

그러나 대상 API로 게시하는 이미지는 상품의 대표 이미지, 추가 이미지를 등록하는 용도로만 설계되어있습니다. 설계 용도 이외의 사용에 대해서는 향후 운용 상황 변동에 따른 영향까지 보장하긴 어려울 수 있습니다.

상품 상세 설명 내용으로 이미지 게시가 필요한 경우 스마트스토어 외 외부 이미지 호스팅을 이용할 수도 있습니다.

**Reply by importunate-dev** [reply]
친절한 답변 감사합니다.
